tools/package-lock.json

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
do we need this file?

Reply:
I removed package-lock.

You might want to consider adding it to you .gitignore